CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## v2.18.3
 
+### `@liveblocks/core`
```

Comment:
`@liveblocks/core` is internal, this should be in `@liveblocks/client` (or everywhere `stringifyCommentBody` is re-exported publicly)